mmdet/core/post_processing/matrix_nms.py

```diff
@@ -0,0 +1,120 @@
+import torch
+
+
+def mask_matrix_nms(masks,
+                    labels,
+                    scores,
+                    update_thr=-1,
+                    nms_pre=-1,
+                    max_num=-1,
+                    kernel='gaussian',
+                    sigma=2.0,
+                    mask_area=None):
+    """Matrix NMS for multi-class masks.
+
+    Args:
+        masks (Tensor): Has shape (num_instances, h, w)
+        labels (Tensor): Labels of corresponding masks,
+            has shape (num_instances,).
+        scores (Tensor): Mask scores in descending order,
```

Comment:
Mask scores in descending order. Why do we need to sort it later？

Reply:
This is copied from the original repo `cate_scores (Tensor): shape (n), mask scores in descending order`